### DISCUSSION TITLE
StackStorm SSO integrated with Azure AD using SAML.

### DISCUSSION THREAD

**Discussion by MarlonHeiber** [StackStorm/st2#5580] [Documentation]
Hey guys, I'm creating this discussion because I think this is the question of many people.

How to do StackStorm SSO integrated with Azure AD using SAML. 

Has anyone managed to do this integration?

On the official page this is in the backlog, and in the RoadMap to be done: https://docs.stackstorm.com/roadmap.html?highlight=sso

Heres's the link of SSO backend : https://github.com/StackStorm/st2-auth-backend-sso-saml2

I didn't find any documentation on how to do this, if anyone has it and can post it here to help the community.

**Comment by setswei** [comment]
I also require SSO functionality, I am hoping PR #5664 will be merged soon
